app/assets/stylesheets/modules/top_bar.scss

```diff
@@ -0,0 +1,40 @@
+@import "variables";
+
+.top-bar {
+ background: $topbar-bg-color;
+
+ .name h1 {
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Selector should have depth of applicability no greater than 2, but was 3